openedx/features/student_account/constants.py

```diff
@@ -1,2 +1,10 @@
 NON_ACTIVE_COURSE_NOTIFICATION = 'The course you enrolled in, %s opened last week but you can start it right now. <a ' \
             'class="click-here-link" href="%s">Click here</a> to get started.'
+
+TOP_REGISTRATION_COUNTRIES = (
+    ('NG', 'Nigeria'),
+    ('IN', 'India'),
+    ('PK', 'Pakistan'),
+    ('KE', 'Kenya'),
+    ('GH', 'Ghana'),
+)
```

Comment:
I am just curious, Can't we use same format as COUNTRIES as on front end both will be used for same element?

Reply:
This was discussed on our call. This is same as the `all_countries` dict element.